content/terraform-plugin-testing/v1.13.x/docs/plugin/testing/testing-patterns.mdx

```diff
@@ -200,6 +201,61 @@ tests is sufficient as opposed to having two separate tests.
 
 [Back to top](#table-of-contents)
 
+## Import mode: verify that import and create produce the same state
+
+To exercise a provider's import behaviors, the test author adds a test step in import mode.
```

Comment:
💭 add hyperlinks for:

- "import behaviors"
- "`ImportState: true`"
- ...

Reply:
I like the idea of breaking this documentation into it's own section at some point. Lots of different pieces to discuss in this mode 😆